js/mexc.js

```diff
@@ -1476,6 +1479,10 @@ module.exports = class mexc extends Exchange {
                 'currency': code,
             };
         }
+        if (type === 'withdrawal') {
+            // mexc withdrawal amount includes the fee
+            amount = amount - feeCost;
```

Comment:
We have to do this with string-based math: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings:

```JavaScript
let amount = this.safeString (transaction, 'amount');
let feeCost = this.safeString (transaction, 'fee');
if (type === 'withdrawal') {
    amount = Precise.stringSub (amount, feeCost);
}
if (feeCost !== undefined) {
    feeCost = this.parseNumber (feeCost);
    ...
}
...
return {
    ...
    'amount': this.parseNumber (amount),
    ...
};
```